Sources/RInAppMessaging/Models/Trigger.swift

```diff
@@ -21,23 +21,30 @@ internal struct Trigger: Codable, Equatable {
 }
 
 internal struct TriggerAttribute: Codable, Equatable {
+    private enum CodingKeys: String, CodingKey {
+        case name
+        case value
+        case type
+        case operatorType = "operator"
+    }
+
     let name: String
     let value: String
     let type: AttributeType
-    let `operator`: AttributeOperator
```

Comment:
nit: SonarQube does not accept `operator`?

Reply:
does not accept names in ` `` `. Anil asked me to change the name if possible (I wanted to make an exception)